File: FCUBEAPI/FreightMasters/Models/ChallanListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FreightMasters.Models
{
    public class ChallanListModel
    {
        public List<ChallanMasterModel> ChallanList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }

    }
}

File: OMKARAUTOAPP/src/app/models/challanlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Challanmastermodel } from "./challanmastermodel";

export class Challanlistmodel {
  challanList: Challanmastermodel[] = [];
  pageMetaData: Paginationmodel = new Paginationmodel;
}
